docs_guidelines/apidocs/tutorials.guidelines.rst

```diff
@@ -0,0 +1,110 @@
+####################
+Tutorials Guidelines
+####################
+
+1. Get the user started
+=======================
+
+1.1 Lower the barrier of entry 
+------------------------------
+
+Lower the barrier of entry as much as possible. Assume the user does not know much about what is
+covered in the tutorial. If we are expecting users to know something, make it explicit and link them
+to the needed explanations (See :ref:`tutorial-guidelines-minimum-explanation`) Avoid jargon as much as possible. Provide a minimum
+necessary explanation when the jargon is introduced. (See :ref:`tutorial-guidelines-minimum-explanation`)
+ 
+1.2 Provide setup / prerequisites
+---------------------------------
+
+1.3 Add sufficient comments to code
+-----------------------------------
+
+2. Provide a complete picture before they start
+===============================================
+
+2.1 Provide overview at the start
+---------------------------------
+
+Summarize what the tutorial is about in the first paragraph, ideally the first sentence
+
+    "In this tutorial, you will..."
+
+2.2 Provide context
+-------------------
+
+Add a context for why you want to do this tutorial? Or who is this tutorial for?
+
+2.3 Summarize at the end
+------------------------
+
+Summarize what the user has accomplished at the end of the tutorial
+
+    "You have built a secure, three-layer hylomorphic stasis engine…" Describe (and admire, in a 
+    mild way) what your learner has accomplished (note - not: “you have learned…”)
+
+3. Ensure that the tutorial works reliably and repeatable
+=========================================================
+
+3.1 Test tutorials regularly
+----------------------------
+
+on supported platform and python / Qiskit versions (for example, via CI
```

Comment:
this reads like a note, i think it should be a full sentence. Also explain what CI is otherwise we aren't practicing what we preach about jargon haha

Reply:
These guidelines are supposed to be mainly for the mantainers of a Qiskit package. I think it's fair to assume they know what CI is. Also don't forget that this is not supposed to be a tutorial (so jargon is not so big a problem) but I can add an explanation about CI. What do you think? @javabster @HuangJunye

Reply:
yeah most of the time i think it's fine to assume that, it's just that i was reading this straight after the section when you talk about not using jargon etc. and it just felt a bit hypocritical 😅

Reply:
i wouldn't block merging on this bc of the acronym, just a suggestion. I do think it should be a full sentence though, and generally i think we should keep the grammar consistent in the page. so either all short note form or all full sentences, not a mix of both